### PR TITLE
Fix Cirrus CI

### PR DESCRIPTION
**Pull request by JesterOrNot**

### CODE REVIEW

**Comment by JesterOrNot** [comment]
@RDIL Take a look

**Comment by RDIL** [comment]
@fkorotkov GitHub checks are still being whacky, any idea why?

File: .cirrus.yml [hunk 1]
```diff
@@ -14,15 +14,10 @@ task:
...
-task:
-  name: Test Docker Image
-  container:
-    image: gcr.io/kaniko-project/executor:v0.9.0
-  env:
-    CIRRUS_SHELL: direct
-    CIRRUS_WORKING_DIR: /workspace
-  build_script: executor --destination=build:local
+docker_builder:
+  build_script: docker build --tag gosh:latest .
```
Comment (RDIL) [inline .cirrus.yml]:
```suggestion
  build_script: docker build --tag gosh:latest .
  name: Docker Build
```
add a name so it doesn't just say `main`

**Comment by fkorotkov** [comment]
@RDIL can you be more specific?

**Comment by RDIL** [comment]
@fkorotkov see screenshot below
Screenshot 2019-12-18 at 4 00 28 PM

**Comment by RDIL** [comment]
It looks like there was a commit to `.gitpod.yml` somewhere that isn't in this PR but that Cirrus CI autocancelled because of? (https://cirrus-ci.com/build/5936274652790784)

**Comment by fkorotkov** [comment]
Not quite sure why GH shows https://github.com/gosh-terminal/gosh/runs/355247763 instead of a later check run https://github.com/gosh-terminal/gosh/runs/355255765

**Comment by fkorotkov** [comment]
Oh. Seems there was a race condition reporting the auto-cancelation. 🤔 Need further investigation. Thanks for reporting and please let me know if you'll see it again. 🙌

**Comment by RDIL** [comment]
@JesterOrNot looks good to go then.

**Comment by JesterOrNot** [comment]
@RDIL merge?

**Comment by RDIL** [comment]
Yeah

**Pull request merged**